### ISSUE TITLE
I have no idea what's wrong

### ISSUE THREAD

**Issue by ghost** [apsun/TumblrAdAway#14]
Write your issue description here.

My issue is well, ads are there. Both from ad pages and the non clickable sponsored ones. I use the alpha versions of Tumblr (that may be my issue there) but I've NEVER had this happen before with any previous alpha. Maybe Tumblr changed their way of doing ads, not sure. I also use a few ad blocking apps that in no way block them either. Any advice you can shoot me? Oh, on my other device I'm using systemless Xposed arm7 on Android 7.1.2 and it doesn't seem to be showing ads on that one. Both devices are set up the same otherwise. Thanks for at least reading this. If I added something wrong, my apologizes, first time actually using GitHub

**If you are submitting a bug report and do not include the following info, your issue will be ignored!**

- Device ("Honor 8"): 
- Android version ("7.0"): 
- Xposed version ("89"): 
- Tumblr version ("10.0.2.14"): 
- Tumblr AdAway version ("1.8"): 

Please paste your Xposed logs (Xposed Installer -> Logs -> Menu -> Save to SD card) below:

```
None showing up
```

Thank you for helping us help you help us all.

**Comment by apsun** [comment]
Yeah, it's possible that they changed the ad code. Unfortunately I don't have access to a phone with Xposed at the moment, so you might have to wait a few weeks for me to fix this, sorry about that. In the meantime you can always downgrade to a previous version of Tumblr.

**Comment by ghost** [comment]
Appreciate the response! It's not a big deal at all for me so I'll gladly wait.

**Comment by apsun** [comment]
Hm, so I just took a look, and the module appears to work fine on the latest Tumblr version (10.0.2) for me (except for a few headers, which I'll get around to fixing). I would advise checking your configuration again to see if it's a problem on your end. Otherwise, it might be an A/B test (this is what happened a few months ago), and there's no telling when/if I will get the changes.
